Source/CharismaModule/Private/PlaythroughStartConversation.cpp

```diff
@@ -0,0 +1,39 @@
+// Fill out your copyright notice in the Description page of Project Settings.
```

Comment:
I totally get where this is coming from, though I think attaching handlers to the `Playthrough` is a distinct step from starting the playthrough so I'm not sure they should be related.

Perhaps instead of a `PlaythroughStartConversation` async node, it could be an `AttachPlaythroughHandlers` async node, which has the `Playthrough` as an input pin, and lots of async output pins, one for each event such as `OnReady`, `OnMessage`, `OnTyping` etc? What do you think?